test/e2e-tests/tests/pipelines_test.go

```diff
@@ -11,9 +11,13 @@ import (
 )
 
 const (
-	AIEdgeE2EPipelineDirectoryRelativePath       = "../../../pipelines/tekton/aiedge-e2e"
-	BikeRentalsPipelineRunFileRelativePath       = AIEdgeE2EPipelineDirectoryRelativePath + "/aiedge-e2e.bike-rentals.pipelinerun.yaml"
-	TensorflowHousingPipelineRunFileRelativePath = AIEdgeE2EPipelineDirectoryRelativePath + "/aiedge-e2e.tensorflow-housing.pipelinerun.yaml"
+	MLOpsPipelineDirectoryRelativePath            = "../../../pipelines/tekton/aiedge-e2e"
+	MLOpsBikeRentalsPipelineRunRelativePath       = MLOpsPipelineDirectoryRelativePath + "/aiedge-e2e.bike-rentals.pipelinerun.yaml"
+	MLOpsTensorflowHousingPipelineRunRelativePath = MLOpsPipelineDirectoryRelativePath + "/aiedge-e2e.tensorflow-housing.pipelinerun.yaml"
```

Comment:
I wonder whether we shouldn't reflect the actual name (aiedge-e2e), but it will likely change anyway...

Reply:
Sure I'll change it. And the git one aswell